install/local/dev/php-fpm/Dockerfile

```diff
@@ -137,4 +137,4 @@ RUN sed -i "s/BASEDOMAIN/${MAIN_DOMAIN}/g" conf/wikijump.ini && \
 
 COPY etc/php/ /usr/local/etc/php/conf.d/
 
-CMD ["php-fpm"]
+ENTRYPOINT ["sh", "-c", "php artisan migrate && php-fpm"]
```

Comment:
`php artisan migrate` handles the database migration and seeding. If the migrations have already been run, nothing happens, so new containers won't inadvertently destroy or reset data.